app/assets/javascripts/app/views/location_stream.js

```diff
@@ -1,6 +1,6 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
-app.views.LocationStream = app.views.Content.extend({
+app.views.LocationStream = app.views.Base.extend({
```

Comment:
@svbergerem, can you add some context without me reading into the details? I expect there is a reason for having the Content view in the prototype chain of our location stream, instead of directly using the base stream?